modules/stochastic_tools/src/distributions/BoostLognormalDistribution.C

```diff
@@ -9,27 +9,18 @@
 
 #include "BoostLognormalDistribution.h"
 
-#include "libmesh/auto_ptr.h"
-
-registerMooseObjectAliased("StochasticToolsApp", BoostLognormalDistribution, "BoostLognormal");
 registerMooseObjectReplaced("StochasticToolsApp",
                             BoostLognormalDistribution,
                             "07/01/2020 00:00",
-                            BoostLognormal);
+                            Lognormal);
 
 InputParameters
 BoostLognormalDistribution::validParams()
 {
-  InputParameters params = Distribution::validParams();
-  params.addClassDescription("Boost Lognormal distribution.");
-  params.addRequiredParam<Real>("location", "The Lognormal location parameter.");
-  params.addRequiredParam<Real>("scale", "The Lognormal scale parameter.");
-  return params;
+  return BoostLognormal::validParams();
 }
 
 BoostLognormalDistribution::BoostLognormalDistribution(const InputParameters & parameters)
-  : BoostDistribution<boost::math::lognormal_distribution<Real>>(parameters)
+  : BoostLognormal(parameters)
 {
-  _distribution_unique_ptr = libmesh_make_unique<boost::math::lognormal_distribution<Real>>(
-      getParam<Real>("location"), getParam<Real>("scale"));
 }
```

Comment:
Is there a reason we are keeping these functions around? It seems to clutter up our distributions folder.

Reply:
We were previously using some trickery with `registerMooseObjectReplaced` to keep the old class and register the new name pointing to the old class. That caused me a lot of trouble when I was trying to use `registerMooseObjectReplaced` on the Boost distributions to point to a class that didn't exist (the one without `Distribution` at the end of the name).

I thought it was easiest to just move the code to the classes without `Distribution` in the name (because those are the ones that we are going to keep, and then make classes with `Distribution` at the end that are simply wrappers around the new classes that make them available with the deprecated names. When all these classes are deprecated on 7/1/2020, we just need to go through and remove all classes with `Distribution` at the end of them and all the `Boost` classes. It does end up with some extra files in the directory, but it's temporary.

Reply:
Oh okay, I guess there is no easy way to replace and alias at the same time. I have 7/1/2020 marked on my calendar now.